### PR TITLE
Check if setting has been set before accessing value to prevent error

### PR DESCRIPTION
**Pull request by peterdietz**
Added a method to lookup setting value that ensures it has been set first

Addresses issue #46

### CODE REVIEW

File: lib.php [hunk 1]
```diff
@@ -812,6 +812,10 @@ function plagiarism_vericite_get_settings() {
     }
 }
 
+function plagiarism_vericite_get_setting_boolean($plagiarismsettings, $key) {
```
Comment (baholladay) [inline lib.php]: this function should take a parameter for the default, not just "false"

**Comment by peterdietz** [comment]
Updated to allow for default value param. Renamed the function to be less wordy

File: lib.php [hunk 2]
```diff
@@ -812,8 +812,8 @@ function plagiarism_vericite_get_settings() {
     }
 }
 
-function plagiarism_vericite_get_setting_boolean($plagiarismsettings, $key) {
-    return isset($plagiarismsettings[$key]) ? $plagiarismsettings[$key] : false;
+function get_setting_boolean($plagiarismsettings, $key, $default = false) {
```
Comment (baholladay) [inline lib.php]: you were correct with the "plagiarism_vericite_" prefix before, this is a requirement to prefix all functions like this in Moodle plugins, please switch back

**Comment by peterdietz** [comment]
Added prefix back to function name.

**Pull request merged**